src/backend/core/api/viewsets.py

```diff
@@ -191,6 +198,142 @@ def get_me(self, request):
         )
 
 
+class StatsViewSet(viewsets.GenericViewSet):
+    """
+    API endpoint that provides various statistics in JSON format.
+    """
+
+    permission_classes = [IsAdminUser]
+
+    def list(self, request):
+        user_timezone = "UTC"
+        if request.user.is_authenticated:
+            user_timezone = str(getattr(request.user, "timezone", user_timezone))
+
+        try:
+            timezone_to_use = ZoneInfo(user_timezone)
+        except ZoneInfoNotFoundError:
+            return drf_response.Response(
+                {"detail": "Invalid user timezone provided."},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        if "tz" in request.query_params:
+            try:
+                timezone_to_use = ZoneInfo(request.query_params["tz"])
+            except ZoneInfoNotFoundError:
+                return drf_response.Response(
+                    {"detail": "Invalid timezone provided."},
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
```

Comment:
We don't need this for the moment, I suggest we keep things simple and don't manage the timezone change from URL now.